sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -1905,4 +1905,15 @@ object QueryExecutionErrors {
   def cannotConvertOrcTimestampToTimestampNTZError(): Throwable = {
     new RuntimeException("Unable to convert timestamp of Orc to data type 'timestamp_ntz'")
   }
+
+  def writePartitionExceedConfigSizeWhenDynamicPartitionError(
+    numWrittenParts: Int,
```

Comment:
nit: we need to use 4-space indentation here.

Reply:
check here: https://github.com/databricks/scala-style-guide#indent

Reply:
Sorry, I did not notice the indentation problem here, you have already provided an example before, thank you.